[functorch] Delete functorch_lagging_op_db

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #83418

No need to have a lagging op db because there are no more sync issues
between functorch and pytorch. If someone adds a new OpInfo, then we
should explicitly check if we support it or not.

zou3519 committed:
[functorch] Delete functorch_lagging_op_db

No need to have a lagging op db because there are no more sync issues
between functorch and pytorch. If someone adds a new OpInfo, then we
should explicitly check if we support it or not.

[ghstack-poisoned]

samdow reviewed (approved):
Looks like a couple other files are still importing `functorch_lagging_op_db` but the idea makes sense

zou3519 committed:
Update on "[functorch] Delete functorch_lagging_op_db"


No need to have a lagging op db because there are no more sync issues
between functorch and pytorch. If someone adds a new OpInfo, then we
should explicitly check if we support it or not.

[ghstack-poisoned]

zou3519 commented:
@pytorchbot merge -f "windows failures are a known problem"
